doc/radosgw/dynamicresharding.rst

```diff
@@ -8,47 +8,50 @@ RGW Dynamic Bucket Index Resharding
 
 A large bucket index can lead to performance problems. In order
 to address this problem we introduced bucket index sharding.
-Until Luminous, changing the number of bucket shards (resharding)
-needed to be done offline, from Luminous we support
+Until Luminous changing the number of bucket shards (resharding)
```

Comment:
I would have left the comma.

Reply:
@adamemerson I'll add the comma back and merge. Thanks